lib/sequent/core/base_event_handler.rb

```diff
@@ -47,5 +47,8 @@ def initialize(record_session = Sequent::Core::RecordSessions::ActiveRecordSessi
                      :find_records, :last_record
 
     end
+
+    # Alias the above class
+    Projector = BaseEventHandler
```

Comment:
Since `BaseEventHandler` now only serves projections and no longer workflows, maybe make `Projector` the main class and have a deprecated alias for `Projector` for backwards compatibility?